_data/communication.yml

```diff
@@ -134,6 +134,7 @@ websites:
       img: sendgrid.png
       tfa: Yes
       sms: Yes
+      software: Yes
       doc: https://sendgrid.com/docs/User_Guide/two_factor_authentication.html
```

Comment:
can you update the `doc` link to https://sendgrid.com/docs/User_Guide/Settings/two_factor_authentication.html?